src/vs/platform/environment/node/argv.ts

```diff
@@ -410,9 +413,12 @@ function indent(count: number): string {
 function wrapText(text: string, columns: number): string[] {
 	const lines: string[] = [];
 	while (text.length) {
-		const index = text.length < columns ? text.length : text.lastIndexOf(' ', columns);
```

Comment:
there was a 10-year-old bug here where this would crash if the last space in the line was the first character!